app/Http/Controllers/FrontendTransportController.php

```diff
@@ -42,6 +42,22 @@ public function TrainStationboard(Request $request) {
         );
     }
 
+    public function StationByCoordinates(Request $request)
+    {
+        $NearestStation = TransportBackend::StationByCoordinates($request->latitude, $request->longitude);
```

Comment:
Can we add [input validation](https://laravel.com/docs/5.8/validation) for this?

Reply:
You can validate the input like this:

```suggestion
    {
	    $request->validate([
	        'latitude' => 'required|numeric|min:-180|max:180',
	        'longitude' => 'required|numeric|min:-90|max:90'
	    ]);
        $NearestStation = TransportBackend::StationByCoordinates($request->latitude, $request->longitude);
```